explorer/ui/src/components/BlockDAG.tsx

```diff
@@ -244,10 +241,18 @@ export class BlockDAG extends React.Component<Props, {}> {
       .append('g')
       .attr('class', 'links')
       .selectAll('line')
-      .data(graph.links)
+      .data(graph.links.filter((d:d3Link) => {
+        if (hideBallot) {
+          return isBlock(d.source.block) && isBlock(d.target.block);
+        } else {
+          return true;
+        }
+      }))
```

Comment:
```suggestion
      .data(graph.links.filter((d:d3Link) => !hideBallot || isBlock(d.source.block) && isBlock(d.target.block)))
```
or maybe 
```
      .data(graph.links.filter((d:d3Link) => hideBallot ? isBlock(d.source.block) && isBlock(d.target.block)) : true)
```
Maybe with multiple cases a full blown if-then-else would be better, but it seems like a lot of lines for something that can be simplified like this. Anyway, up to you.